common/metrics/defs.go

```diff
@@ -430,7 +439,7 @@ var ScopeDefs = map[ServiceIdx]map[int]scopeDefinition{
 		TransferQueueProcessorScope:                 {operation: "TransferQueueProcessor"},
 		TransferTaskActivityScope:                   {operation: "TransferTaskActivity"},
 		TransferTaskDecisionScope:                   {operation: "TransferTaskDecision"},
-		TransferTaskDeleteExecutionScope:            {operation: "TransferTaskDeleteExecution"},
+		TransferTaskCloseExecutionScope:             {operation: "TransferTaskCloseExecution"},
```

Comment:
Any specific reason to rename this?

Reply:
this is no longer deleting the execution, it is just close it and copy data to visibility store. So I think the name with delete is confusing.

Reply:
Got it.